hooks/useSharedGuideIds.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  useState,
+  useEffect,
+  useContext,
+  createContext,
+  ReactNode,
+  Dispatch,
+  SetStateAction,
+} from "react";
+import { SHARED_GUIDES_LOCAL_STORAGE } from "utils/constants";
+
+const SHARED_GUIDES_INITIAL_STATE = [""];
+
+interface SharedGuideIdsContextResult {
+  sharedGuideIds: string[];
+  setSharedGuideIds: Dispatch<SetStateAction<string[]>>;
+}
```

Comment:
I'm wondering about the `SharedGuides` name... I feel like `AccessedVotingGuides` or `SavedVotingGuides` or something along those lines would be more indicative of what this list is

Reply:
Agreed

Reply:
I'm going with `SavedVotingGuides`